stdlib/distutils/command/bdist_wininst.pyi

```diff
@@ -0,0 +1,19 @@
+import sys
+from _typeshed import StrOrBytesPath
+from typing import Any, ClassVar
+
+if sys.version_info < (3, 10):
```

Comment:
Instead of using this guard, the file should be added to `VERSIONS`.

Reply:
It is already there: https://github.com/python/typeshed/blob/5c309e5fe4a8d4898fe9f2b8664bb66cff845878/stdlib/VERSIONS#L116

Reply:
I'm not sure I see the comments you're referring to, but cf https://github.com/python/mypy/pull/12083

Reply:
https://github.com/python/typeshed/pull/7249#issuecomment-1042909330